proto/agent/v1/connection.proto

```diff
@@ -30,6 +30,8 @@ message UpdateAgentConnection {
 
     // How long the netdata agent was running until connection (only applicable when reachable=true)
     google.protobuf.Duration agent_uptime = 16;
+
+    repeated aclk_lib.v1.Capability capabilities = 17;
```

Comment:
[minor] Adding the `capabilities` field on this message I think is ok, but I'm not sure if the `enabled` field of the capability has any interpretation here, as some capabilities can be enabled per node.